homeassistant/components/moehlenhoff_alpha2/translations/de.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
this file is created once it goes in and it gets translated, can be deleted from the PR

Reply:
Deleted.